### PR TITLE
Update 'Create and debug Office Add-in in Visual Studio' article

### PR DESCRIPTION
**Pull request by kbrandl**
Addresses issues #222 and #324.

**_Note_**: Given the scope of changes I've made to this article, it's probably easiest to review by looking at the updated article on the staging site (instead of just looking at the file diff in this PR): https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/create-and-debug-office-add-ins-in-visual-studio?branch=kb-update-VS-article-222#debug-your-add-in.

### CODE REVIEW

File: docs/develop/create-and-debug-office-add-ins-in-visual-studio.md [hunk 1]
```diff
@@ -1,122 +1,145 @@
...
 
-Start the solution from the menu bar by choosing **Debug** > **Start Debugging**. Visual Studio will automatically build the solution and start Office to host your add-in.
+### Start the project
+
+Start the project by choosing **Debug** > **Start Debugging** from the menu bar. Visual Studio will automatically build the solution and start Office to host your add-in.
+
+> [!NOTE]
+> When you start an Outlook add-in project, you'll be prompted for login credentials. If you're asked to log in repeatedly then Basic Auth may be disabled for accounts on your Office 365 tenant. In this case, try using a Microsoft account instead.
 
 When Visual Studio builds the project it performs the following tasks:
```
Comment (davidchesnut) [inline docs/develop/create-and-debug-office-add-ins-in-visual-studio.md]: I just noticed that in the step 1, it says the manifest is copied to the ProjectName\Output directory. But when I run VS, it puts the manifest in bin\debug\OfficeappManifests folder. Need to confirm, but this info might be out of date.

File: docs/develop/create-and-debug-office-add-ins-in-visual-studio.md [hunk 2]
```diff
@@ -1,122 +1,145 @@
...
-|Web application project|Contains the content pages of your add-in, including all the files and file references that you need to develop Office-aware HTML and JavaScript pages. While you develop your add-in, Visual Studio hosts the web application on your local IIS server. When you're ready to publish, you'll have to find a server to host this project. To learn more about ASP.NET web application projects, see [ASP.NET Web Projects](http://msdn.microsoft.com/library/cdcd712f-96b0-4165-8b5d-9d0566650a28%28Office.15%29.aspx).|
+3. Name the project, and then choose **OK**.
 
-## Modify your add-in settings
+4. If you've chosen to create a **Word Web Add-in** or an **Outlook Web Add-in**, Visual Studio creates a solution and its two projects appear in **Solution Explorer**. Proceed to the next section of this article to [explore the Visual Studio solution](#explore-the-visual-studio-solution).
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/develop/create-and-debug-office-add-ins-in-visual-studio.md]: Good suggestion; I'll make this change.

File: docs/develop/create-and-debug-office-add-ins-in-visual-studio.md (hunk 1 above)
Comment (kbrandl) [reply docs/develop/create-and-debug-office-add-ins-in-visual-studio.md]: Good catch! I've updated this section accordingly.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-VS-article-222/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811022040312508-538%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RxWjkQom5VtFMVz0XFviGqMCeSFnfVy2W8WcbdOi4bs%3D&st=2018-11-02T20%3A36%3A46Z&se=2018-12-03T20%3A41%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-update-VS-article-222/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811022045176456-538%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kISJNaZ51EFumpiuWB%2BVjIBMtw6%2BThsrqWMbd%2BfsTbo%3D&st=2018-11-02T20%3A41%3A31Z&se=2018-12-03T20%3A46%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**